src/Postnl.php

```diff
@@ -329,6 +337,42 @@ public function currentStatus($barcode)
     }
 
     /**
+     * @param string $postalCode
+     * @param ComplexTypes\ArrayOfCutOffTime $cutOffTimes
+     * @param string $shippingDate
+     * @param int $shippingDuration
+     * @param string[] $options
+     * @param string $allowSundaySorting
+     * @param string $countryCode
+     */
+    public function getDeliveryDate(
+        $postalCode,
+        $cutOffTimes,
+        $shippingDate,
+        $shippingDuration = 1,
+        $options = ['Daytime'],
+        $allowSundaySorting = 'false',
+        $countryCode = 'NL'
+    ) {
+        /** @var ComplexTypes\GetDeliveryDate $GetDeliveryDate */
```

Comment:
Oh, just typehinting. PHPStorm didn't understand the return value for that magical ``create`` method, so it couldn't autocomplete for $GetDeliveryDate variable. I can remove it if you want. It's just developer convenience.